src/clarity-angular/data/datagrid/providers/selection.spec.ts

```diff
@@ -114,6 +114,33 @@ export default function(): void {
             expect(selectionInstance.isSelected(4)).toBe(false);
         });
 
+        it("handles clrDgItems trackBy index expressions in multi selection type", function() {
+            // IMPORTANT, using clrDgItems with trackBy will use trackBy reference in multi
+            selectionInstance.selectionType = SelectionType.Multi;
+            itemsInstance.trackBy = (index, item) => index;
+            // Must set the trackBy before setting the items
+            itemsInstance.all = ["a", "b", "c", "d", "e"];
```

Comment:
If you really want to test `trackBy`, you should test with several objects that are equal but the trackBy can differentiate them. Something like `["a", "a", "a", "b", "b"]`.